### PR TITLE
update: adapt changes in multiDeclRefactor.

### PR DESCRIPTION
**Pull request by CharlesZ-Chen**
Small update on the `fetch_map2anno_tools.sh` due to the updates in my `multiDeclRefactor`.

### CODE REVIEW

File: map2annotation/fetch_map2anno_tools.sh [hunk 1]
```diff
@@ -1,17 +1,10 @@
 #!/bin/bash
 MAP2ANNO_DIR=$(cd $(dirname "$0") && pwd)
 
-##### build hacked checkstyle
```
Comment (mernst) [inline map2annotation/fetch_map2anno_tools.sh:4]: Can you explain how the hacked checkstyle is related to the multiDeclRefactor?  Why are these both in the same code review?

Comment (CharlesZ-Chen) [reply map2annotation/fetch_map2anno_tools.sh:4]: The `hacked checkstyle` works as a back-end to `multiDeclRefactor`.  `hacked checkstyle` detects multi-declarations problems and send diagnostic reports to `multiDeclRefactor`, and then `multiDeclRefactor` do refactors based on the reports.

So in this PR, I changed `hacked checkstyle` as a dependency to `multiDeclRefactor`, and move `checkstyle` inside to `multiDeclRefactor`.

Comment (CharlesZ-Chen) [reply map2annotation/fetch_map2anno_tools.sh:4]: The `hacked checkstyle` works as a back-end to `multiDeclRefactor`.  `hacked checkstyle` detects multi-declarations problems and sends diagnostic reports to `multiDeclRefactor`, and then `multiDeclRefactor` do refactors based on the reports.

So in this PR, I changed `hacked checkstyle` as a dependency to `multiDeclRefactor`, and moved `checkstyle` inside to `multiDeclRefactor`.

The benefit of this change would be I can switch backend of `multiDeclRefactor` without influence on the clients of `multiDeclRefactor`, i.e. now `multiDeclRefactor` manages the backend dependencies by itself.

Comment (wmdietl) [reply map2annotation/fetch_map2anno_tools.sh:4]: Changes look ok to me.

**Comment by martinschaef** [comment]
Hmm ... looks like I wasn't subscribed to this. @wmdietl @tim-mccarthy is this still current?

**Comment by CharlesZ-Chen** [comment]
@martinschaef  Hi Martin, yes, this PR is still current as the changes in `MultiDeclRefactor`.

**Pull request merged**